parsers/US_PJM.py

```diff
@@ -220,18 +220,18 @@ def get_miso_exchange(session: Session) -> tuple:
         raise ValueError("US-MISO->US-PJM flow direction cannot be determined.")
 
     find_timestamp = soup.find("div", {"id": "body_0_divTimeStamp"})
-    dt_naive = parser.parse(find_timestamp.text)
-    dt_aware = add_default_tz(dt_naive)
+    dt = parser.parse(find_timestamp.text, tzinfos={"EDT": -4 * 3600, "EST": -5 * 3600})
+    if _is_naive(dt):
+        dt = dt.replace(tzinfo=TIMEZONE)
```

Comment:
note: `dt_naive` wasn't really naive as it includes EDT / EST information. 

I was getting
```
UnknownTimezoneWarning: tzname EDT identified but not understood. 
Pass `tzinfos` argument in order to correctly return a timezone-aware datetime.
In a future version, this raise an exception.
category=UnknownTimezoneWarning)
```

The new code should make sure that those are mapped to known utc offsets. The old code was remapping everything to "America/New_York" so I think it could have potentially been assigning the wrong UTC offset if e.g. the timestamp was marked as EDT and New York was observing EST (although I imagine that in practice - at least for live data - those would always be in sync)

Just in case there are really naive timestamps coming through I have kept the fallback to make sure those would get assigned the correct timezone.